styles.css

```diff
@@ -47,3 +50,22 @@
 #item-f {
     grid-area: block-f;
 }
+
+/*CSS-Grid settings for tablets and bigger screens*/
+@media screen and (min-width: 576px) {

```

Comment:
Alleen properties die overschreven moeten worden zijn opnieuw gedifinieerd in deze breakpoint.